.github/workflows/deploy.yml

```diff
@@ -0,0 +1,67 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ 'development', 'master' ]
+  pull_request:
+    branches: [ 'development' ]
+
+jobs:
+  cancel-previous:
+    name: 'Cancel Previous Runs'
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          access_token: ${{ github.token }}
+
+  deploy:
+    runs-on: ubuntu-latest
+    needs: cancel-previous
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: '11.15.0'
+          registry-url: https://registry.npmjs.org/
+
+      - name: Configure Git
+        run: |
+          git config --global user.email ${{ secrets.GH_EMAIL }}
+          git config --global user.name ${{ secrets.GH_USER }}
```

Comment:
What did you end up using for `GH_EMAIL` and `GH_USER`? If secret we can message

Reply:
GH_USER : `energyweb`
GH_EMAIL: `devops@energyweb.org`
There is possibility, it might fail. Let's see.